index.js

```diff
@@ -12,16 +17,17 @@ const slugs = new Slugger()
 
 /**
  * Plugin to add `id`s to headings.
- *
- * @type {import('unified').Plugin<Array<void>, Root>}
+ * @type {import('unified').Plugin<[Options?]|Array<void>, Root>}
  */
-export default function rehypeSlug() {
+export default function rehypeSlug(options = {}) {
+  let prefix = options.prefix || '';
```

Comment:
```suggestion
  const prefix = options.prefix || ''
```

It’s used once so `const`. We don‘t use semicolons.